app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
+  has_one :picture, as: :picturable
+
+  enum status: { public: 1, private: 2 }, _prefix: true
```

Comment:
It generates `status_public` and `status_private` with `_prefix: true` option.

`Post.public_status` -> _suffix: true
Consider using second option